lib/ansible/modules/windows/win_updates.ps1

```diff
@@ -86,6 +88,39 @@ $update_script_block = {
             return $result
         }
 
+        Write-DebugLog -msg "Setting Windows Update Agent search source..."
+        try {
+            switch ($server_selection) {
+                "default" {
+                    $server_selection_value = 0
+                    $result.server_selection = "default"
```

Comment:
We try and shy away from setting return values that are just the same as the input module options. In devel/2.8 we've added a framework for modules to do this automatically but not all modules have been moved across to it yet. Taking this into consideration you could just do

```
$server_selection_value = switch ($server_selection) {
    "default" { 0 }
    "managed_server" { 1 }
    "windows_update" { 2 }
}
```